src/libraries/System.Runtime.Numerics/tests/BigIntegerTests.GenericMath.cs

```diff
@@ -928,6 +928,20 @@ public static void MaxTest()
             Assert.Equal(UInt64MaxValue, NumberHelper<BigInteger>.Max(UInt64MaxValue, 1));
         }
 
+        [Fact]
+        public static void MaxNumberTest()
```

Comment:
Only comparing to `1` in each of these tests might not lead to proper coverage, but as long as the `1` input follows a normal code path in each method, this should be fine. For example, if `1` was handled with some edge case in `BigInteger.MaxNumber` then this would raise an eyebrow, but it doesn't.

Reply:
Yeah, there's going to be more tests added overall after the main feature work goes in.

This is more a "minimal subset" to ensure good coverage of the APIs and the normal edge cases.